### PR TITLE
Add organRIN to experimentalData

### PR DESCRIPTION
**Pull request by cw-dvr8**

### CODE REVIEW

File: terms/experimentalData/organRIN.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "sage.annotations-experimentalData.organRIN-0.0.1",
+    "type": "number",
+    "description": "Organ RIN, as previously measured on organ or region"
```
Comment (karawoo) [inline terms/experimentalData/organRIN.json:5]: I'm not clear on what this means

Comment (cw-dvr8) [reply terms/experimentalData/organRIN.json:5]: @karawoo If I change it to "Organ RNA Integrity Number, as previously measured on organ or region", would that clarify it?

Comment (karawoo) [reply terms/experimentalData/organRIN.json:5]: It's the "as previously measured on organ or region" that I don't really understand. "previously" makes it sound like maybe it has changed or become out of date? and is "region" here brain region -- if so we might want to clarify that.

Comment (cw-dvr8) [reply terms/experimentalData/organRIN.json:5]: I talked to @amapeters, and she said that it's asking if there is any RIN information that has been previously measured on the tissue or brain region and that it's a measure of the quality of the tissue the specimen came from that may indicate other issues.  She is going to look at how it was explained in other consortia.

**Comment by cw-dvr8** [comment]
@amapeters Have you had a chance to look into how this key is being explained in other consortia?

File: terms/qc/organRIN.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "sage.annotations-qc.organRIN-0.0.1",
+    "type": "number",
+    "description": "Organ RIN, as previously measured on organ or region"
```
Comment (Aryllen) [inline terms/qc/organRIN.json:5]:
```suggestion
    "description": "A general quality metric of the organ or tissue, not necessarily a direct RIN measure of the sample used in data generation"
```

**Review by Aryllen — CHANGES_REQUESTED** [review]
Just need to update the description based on @amapeters feedback.

**Review by Aryllen — APPROVED** [review]
Looks good to me!

**Pull request merged**